### PR TITLE
🌐 Add Spanish translation for Deployment - Deploy manually

### PR DESCRIPTION
**Pull request by pedabraham**
Hello 👋!,
In this Pull Request I made the Spanish translation of **deployment/manually.md**

##### This pull request continues with the Spanish translation of #1218

### CODE REVIEW

File: docs/es/docs/deployment/manually.md [hunk 1]
```diff
@@ -65,10 +65,10 @@ And run your application the same way you have done in the tutorials, but withou
...
+Tal vez quedrás configurar algunas herramientas, para asegurarte que se reinicie automáticamente si llega a parar.
 
-You might also want to install <a href="https://gunicorn.org/" class="external-link" target="_blank">Gunicorn</a> and <a href="https://www.uvicorn.org/#running-with-gunicorn" class="external-link" target="_blank">use it as a manager for Uvicorn</a>, or use Hypercorn with multiple workers.
+También quedrás instalar <a href="https://gunicorn.org/" class="external-link" target="_blank">Gunicorn</a> y <a href="https://www.uvicorn.org/#running-with-gunicorn" class="external-link" target="_blank">usarlo como administrador de Uvicorn</a>, o usar Hypercorn con multiples workers.
 
-Making sure to fine-tune the number of workers, etc.
+Asegurándote de ajustar el numero de workers, etc.
 
-But if you are doing all that, you might just use the Docker image that does it automatically.
+Pero si estas haciendo todo eso, podrías solamente usar la imagen de Docker que hace todo automáticamente.
```
Comment (gardenunez) [inline docs/es/docs/deployment/manually.md:74]: Use `estás` with an accent mark.

File: docs/es/docs/deployment/manually.md [hunk 2]
```diff
@@ -65,10 +65,10 @@ And run your application the same way you have done in the tutorials, but withou
 
     </div>
 
-You might want to set up some tooling to make sure it is restarted automatically if it stops.
+Tal vez quedrás configurar algunas herramientas, para asegurarte que se reinicie automáticamente si llega a parar.
 
-You might also want to install <a href="https://gunicorn.org/" class="external-link" target="_blank">Gunicorn</a> and <a href="https://www.uvicorn.org/#running-with-gunicorn" class="external-link" target="_blank">use it as a manager for Uvicorn</a>, or use Hypercorn with multiple workers.
+También quedrás instalar <a href="https://gunicorn.org/" class="external-link" target="_blank">Gunicorn</a> y <a href="https://www.uvicorn.org/#running-with-gunicorn" class="external-link" target="_blank">usarlo como administrador de Uvicorn</a>, o usar Hypercorn con multiples workers.
```
Comment (gardenunez) [inline docs/es/docs/deployment/manually.md:70]: I think `quedrás` is not a valid Spanish word. I would use instead `desee` or `quieras`.

File: docs/es/docs/deployment/manually.md [hunk 3]
```diff
@@ -65,10 +65,10 @@ And run your application the same way you have done in the tutorials, but withou
...
 
-Making sure to fine-tune the number of workers, etc.
+Asegurándote de ajustar el numero de workers, etc.
```
Comment (gardenunez) [inline docs/es/docs/deployment/manually.md:72]: I would use `Asegúrese` instead of `Asegurándote`

File: docs/es/docs/deployment/manually.md [hunk 4]
```diff
@@ -65,10 +65,10 @@ And run your application the same way you have done in the tutorials, but withou
 
     </div>
 
-You might want to set up some tooling to make sure it is restarted automatically if it stops.
+Tal vez quedrás configurar algunas herramientas, para asegurarte que se reinicie automáticamente si llega a parar.
```
Comment (gardenunez) [inline docs/es/docs/deployment/manually.md:68]: I think `quedrás` is not a valid Spanish word. I would use instead `desee` or `quieras`.

File: docs/es/docs/deployment/manually.md [hunk 5]
```diff
@@ -19,13 +19,13 @@ You just need to install an ASGI compatible server like:
     </div>
 
     !!! tip
-        By adding the `standard`, Uvicorn will install and use some recommended extra dependencies.
+        Al añadir el `standard`, Uvicorn instalara y usara algunas dependencias extra recomendadas.
```
Comment (gardenunez) [inline docs/es/docs/deployment/manually.md:22]: A couple of accent marks are missing: `... instalará y usará algunas dependencias ...`

File: docs/es/docs/deployment/manually.md [hunk 6]
```diff
@@ -1,12 +1,12 @@
-# Deploy manually
+# Haz el Deploy manualmente
```
Comment (gardenunez) [inline docs/es/docs/deployment/manually.md:1]: I would use the expression: `Despliegue manualmente` instead.

**Review by gardenunez — COMMENTED** [review]
I have suggested some changes to the translation.

File: docs/es/docs/deployment/manually.md (hunk 3 above)
Comment (pedabraham) [reply docs/es/docs/deployment/manually.md]: I change `Asegurándote` for `Asegurándose` to preserve the original tense

**Comment by pedabraham** [comment]
> I have suggested some changes to the translation.

@gardenunez thank you so much for much for the comments. I made the proper corrections.

**Comment by gardenunez** [comment]
> > I have suggested some changes to the translation.
> 
> @gardenunez thank you so much for much for the comments. I made the proper corrections.

My pleasure to help

File: docs/es/docs/deployment/manually.md [hunk 7]
```diff
@@ -1,12 +1,12 @@
...
 
-You can deploy **FastAPI** manually as well.
+De igual forma puedes desplegar **FastAPI** de forma manual.
```
Comment (jovicon) [inline docs/es/docs/deployment/manually.md:3]: I would use: "También puedes desplegar FastAPI de forma manual"

File: docs/es/docs/deployment/manually.md [hunk 8]
```diff
@@ -1,12 +1,12 @@
...
 
-You just need to install an ASGI compatible server like:
+Solo necesitas instalar un servidor ASGI compatible, como:
 
 === "Uvicorn"
 
-    * <a href="https://www.uvicorn.org/" class="external-link" target="_blank">Uvicorn</a>, a lightning-fast ASGI server, built on uvloop and httptools.
+    * <a href="https://www.uvicorn.org/" class="external-link" target="_blank">Uvicorn</a>, un super rápido servidor ASGI, construido en uvloop y httptools.
```
Comment (jovicon) [inline docs/es/docs/deployment/manually.md:9]: un super rápido servidor ASGI, construido sobre

File: docs/es/docs/deployment/manually.md (hunk 4 above)
Comment (jovicon) [inline docs/es/docs/deployment/manually.md]: you can change: `si llega a parar` for  `si se llega a detener`

File: docs/es/docs/deployment/manually.md (hunk 2 above)
Comment (jovicon) [inline docs/es/docs/deployment/manually.md]: `También quedrás instalar` is not a correct translation for `You might also want to install`
You could change it for: `También podrías querer instalar`

**Review by jovicon — COMMENTED** [review]
I suggested some changes to the translation. @pedabraham

File: docs/es/docs/deployment/manually.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+
+    ```console
+    $ hypercorn main:app --bind 0.0.0.0:80
+
+    Running on 0.0.0.0:8080 over http (CTRL + C to quit)
+    ```
+
+    </div>
+
+Tal vez desearás configurar algunas herramientas, para asegurarte que se reinicie automáticamente si llega a parar.
```
Comment (pedabraham) [inline docs/es/docs/deployment/manually.md:68]:
```suggestion
Tal vez desearás configurar algunas herramientas, para asegurarte que se reinicie automáticamente si se llega a detener.
```

File: docs/es/docs/deployment/manually.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
+# Despliegue manualmente
+
+De igual forma puedes desplegar **FastAPI** de forma manual.
+
+Solo necesitas instalar un servidor ASGI compatible, como:
+
+=== "Uvicorn"
+
+    * <a href="https://www.uvicorn.org/" class="external-link" target="_blank">Uvicorn</a>, un super rápido servidor ASGI, construido en uvloop y httptools.
```
Comment (pedabraham) [inline docs/es/docs/deployment/manually.md:9]:
```suggestion
    * <a href="https://www.uvicorn.org/" class="external-link" target="_blank">Uvicorn</a>, un super rápido servidor ASGI, construido sobre uvloop y httptools.
```

File: docs/es/docs/deployment/manually.md [hunk 11]
```diff
@@ -0,0 +1,74 @@
+# Despliegue manualmente
+
+De igual forma puedes desplegar **FastAPI** de forma manual.
```
Comment (pedabraham) [inline docs/es/docs/deployment/manually.md:3]:
```suggestion
También forma puedes desplegar **FastAPI** de forma manual.
```

**Comment by pedabraham** [comment]
Thank you @jovicon for your review

**Review by bezaca — APPROVED** [review]

**Review by jovicon — APPROVED** [review]

**Comment by tiangolo** [comment]
Thanks @pedabraham! 🤓 

As the original in English has a lot of new and different information, e.g. deployment concepts, I'm gonna pass on this one. But thanks for the work! ☕

**Pull request closed without merging**